src/backend/src/controllers/organizations.controller.ts

```diff
@@ -51,4 +52,18 @@ export default class OrganizationsController {
       next(error);
     }
   }
+
+  static async setLogoImage(req: Request, res: Response, next: NextFunction) {
+    try {
+      const logoFile = req.file || null;
```

Comment:
I don't think you need this line - if `req.file` is not there it will be `undefined `which is also falsy